src/test/java/org/opentripplanner/routing/graph/GraphSerializationTest.java

```diff
@@ -112,7 +112,7 @@ private static void assertNoDifferences (Graph g1, Graph g2) {
         // Skip incoming and outgoing edge lists. These are unordered lists which will not compare properly.
         // The edges themselves will be compared via another field, and the edge lists are reconstructed after deserialization.
         // Some tests re-build the graph which will result in build times different by as little as a few milliseconds.
-        objectDiffer.ignoreFields("incoming", "outgoing", "buildTime");
+        objectDiffer.ignoreFields("incoming", "outgoing", "buildTime", "transitLayer");
```

Comment:
Why is the transit layer ignored when comparing the two graphs? I guess currently it's a derived value, based on the old-style transit object graph. When we remove that object graph, we'll have to remember to re-activate the comparison of the transitLayer - otherwise we're not actually verifying that the transit data is preserved.

Reply:
The idea was that the transit layer was created from the main graph, so as long as the main graph was correct the transit layer was too. But you are right that the transit layer or whatever data structure we end up with should be compared in the test.